test/unittests/skills/test_skill_manager.py

```diff
@@ -111,10 +111,10 @@ def test_instantiate(self):
             'mycroft.skills.all_loaded',
             'mycroft.network.connected',
             'mycroft.internet.connected'
-        ]
-        self.assertListEqual(
+        }
+        self.assertSetEqual(
```

Comment:
dont we want to test order also?

Reply:
Does order matter for bus events? I thought this was just checking which events are registered during init

Reply:
I'd argue they matter, It reflects the order methods are called, but testing that was a side effect and its ok to deprecate.

Reply:
Fair enough, changed that back